app/src/main/java/org/mozilla/focus/Components.kt

```diff
@@ -146,7 +147,8 @@ class Components(
                 AdsTelemetryMiddleware(adsTelemetry),
                 BlockedTrackersMiddleware(context),
                 MergeTabsMiddleware(context),
-                RecordingDevicesMiddleware(context)
+                RecordingDevicesMiddleware(context),
+                CfrMiddleware(context.components),
```

Comment:
Do we want existing users to see these CFRs as well? If yes, we should put this middleware behind a feature flag too. We we release this code without the flag enabled, the settings code in the middleware would continue to increment up. So when the flag is flipped, this CFR would potentially not show up because existing users would have passed that criteria.